src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -529,9 +590,156 @@ class OPENASSETIO_CORE_EXPORT Manager {
   void resolve(const EntityReferences& entityReferences, const trait::TraitSet& traitSet,
                const ContextConstPtr& context, const ResolveSuccessCallback& successCallback,
                const BatchElementErrorCallback& errorCallback);
+
   /**
-   * @}
+   * Provides a @fqref{TraitsData} "TraitsData" populated with the
+   * available data for the requested set of traits for the given @ref
+   * entity_reference.
+   *
+   * @see @ref resolve(const EntityReferences&, <!--
+   * --> const trait::TraitSet&, const ContextConstPtr&, <!--
+   * --> const ResolveSuccessCallback&, <!--
+   * --> const BatchElementErrorCallback& errorCallback)
+   * "Callback signature" variation for more details.
```

Comment:
[fix] 'See's are reworded when rendered, not sure if the grammar works:
![image](https://user-images.githubusercontent.com/12052093/229054675-25570358-ea12-4bf7-9b07-6de402cbbc44.png)

Reply:
Ah sorry, got ahead of myself and squashed before linking you the fixup commit.
built html docs if you wanna see em : [html_docs.zip](https://github.com/OpenAssetIO/OpenAssetIO/files/11121071/html_docs.zip)

Went with : 
```cpp
   * See documentation for the <!--
   * --> @ref resolve(const EntityReferences&, <!--
   * --> const trait::TraitSet&, const ContextConstPtr&, <!--
   * --> const ResolveSuccessCallback&, <!--
   * --> const BatchElementErrorCallback& errorCallback)
   * "callback variation" for more details on resolution behaviour.
```

For all instances of the above pattern on the resolve methods.